py/kc-compat.py

```diff
@@ -39,11 +39,7 @@ def _get_last_byte_from(filename):
     :return: last byte in a file as unsigned int or None if file was empty
     """
     with open(filename, 'rb') as f:
-        try:
-            f.seek(-1, os.SEEK_END)
-        except IOError:
-            return None
-        last, = struct.unpack("B", f.read(1))
+        last, = struct.unpack("B", f.read()[-1])
```

Comment:
do we sure that this file is not too big?

Reply:
Pretty sure. It's always 5 bytes.
I've changed this part due to impossible doing `seek` in efivar files: it raises `OSError: Illegal seek` on any attempts to do any seek.
Thanks!